backend/config.js

```diff
@@ -45,7 +45,7 @@ const config = {
     url: configData.SENTRY_DSN,
   },
   geo: {
-    googleMapsApiKey: configData.GOOGLE_MAPS_API_KEY
+    googleMapsApiKey: configData.GOOGLE_MAPS_API_KEY,
```

Comment:
Ohh this is cool! That missing trailing comma that my local env always misses...